test/readers/test_opencv_reader.py

```diff
@@ -89,7 +89,7 @@ def test_should_start_frame_number_from_two(self):
     def test_should_start_frame_number_from_two_and_offset_from_one(self):
         video_loader = OpenCVReader(
             file_url=os.path.join(PATH_PREFIX, 'dummy.avi'),
-            batch_size=NUM_FRAMES,
+            batch_mem_size=NUM_FRAMES,
```

Comment:
Does all these testcases make sense when we have batch_mem_size? They seem to be wield.

Reply:
You are right. I can get rid of them.

Reply:
Redesigned the test cases to use batch_mem_size.